src/core-plugins/CoreCommands.ts

```diff
@@ -10,7 +11,7 @@ export default class CoreCommands extends Plugin {
         const ClydeUtils = getByProps("sendBotMessage");
         this.commands.registerCommand({
             name: "echo",
-            description: "Creates Clyde message",
+            description: "Creates Clyde message.",
```

Comment:
If you are going to change this, you might as well just fix the grammar too
```suggestion
            description: "Creates a Clyde message.",
```